### PR TITLE
Add param doc isochrones

### PR DESCRIPTION
**Pull request by ZoeBrunet**

### CODE REVIEW

**Comment by ZoeBrunet** [comment]
@stifoon

**Comment by pbougue** [comment]
Ahahah, @stifoon ne suit absolument pas les pokes github :)

**Comment by TeXitoi** [comment]
Does the jsfiddle be updated at the same time?

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -1045,10 +1045,27 @@ You just have to verify that the coordinates of the geocoded object are inside t
...
+| nop       | max_duration            | int           | Maximum duration of isochrones in secondes                                                |               |
+| nop       | min_duration            | int           | Minimum duration of isochrones in secondes                                                |     0         |
+| nop       | <a name="traveler-type"></a>traveler_type | enum | Define speeds and accessibility values for different kind of people.<br>Each profile also automatically determines appropriate first and last section modes to the covered area. Note: this means that you might get car, bike, etc fallback routes even if you set `forbidden_uris[]`! You can overload all parameters (especially speeds, distances, first and last modes) by setting all of them specifically. We advise that you don't rely on the traveler_type's fallback modes (`first_section_mode[]` and `last_section_mode[]`) and set them yourself.<br><div data-collapse><p>enum values:</p><ul><li>standard</li><li>slow_walker</li><li>fast_walker</li><li>luggage</li></ul></div>| standard      |
```
Comment (TeXitoi) [inline documentation/slate/source/includes/apis.md:1050]: Maybe just a "see description in journeys" or something like that? @stifoon ?

Comment (TeXitoi) [inline documentation/slate/source/includes/apis.md:1050]: I don't think the default is standard, I think standard is different from without traveler_type.

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -1045,10 +1045,27 @@ You just have to verify that the coordinates of the geocoded object are inside t
...
 | nop       | last_section_mode[]     | array of string   | Same as first_section_mode but for the last section  | walking     |
 
+### Other parameters
+
+| Required | Name            | Type    | Description                   | Default value |
+|----------|-----------------|---------|-------------------------------|---------------|
+| nop     | max_duration_to_pt   | int     | Maximum allowed duration to reach the public transport.<br>Use this to limit the walking/biking part.<br>Unit is seconds | 15*60 s    |
```
Comment (TeXitoi) [inline documentation/slate/source/includes/apis.md:1060]: default is out of date

File: documentation/slate/source/includes/apis.md [hunk 3]
```diff
@@ -1045,10 +1045,27 @@ You just have to verify that the coordinates of the geocoded object are inside t
...
+| nop     | walking_speed        | float   | Walking speed for the fallback sections<br>Speed unit must be in meter/seconds         | 1.12 m/s<br>(4 km/h)<br>*Yes, man, they got the metric system* |
+| nop     | bike_speed           | float   | Biking speed for the fallback<br>Speed unit must be in meter/seconds | 4.1 m/s<br>(14.7 km/h)   |
+| nop     | bss_speed            | float   | Speed while using a bike from a bike sharing system for the fallback sections<br>Speed unit must be in meter/seconds | 4.1 m/s<br>(14.7 km/h)    |
+| nop     | car_speed            | float   | Driving speed for the fallback sections<br>Speed unit must be in meter/seconds         | 16.8 m/s<br>(60 km/h)   |
+| nop     | max_nb_tranfers      | int     | Maximum number of transfers in each journey  | 10          |
+| nop     | disruption_active    | boolean | For compatibility use only.<br>If true the algorithm take the disruptions into account, and thus avoid disrupted public transport.<br>Rq: `disruption_active=true` = `data_freshness=realtime` <br>Use `data_freshness` parameter instead       |  False     |
```
Comment (TeXitoi) [inline documentation/slate/source/includes/apis.md:1066]: this should not be documented

File: documentation/slate/source/includes/apis.md (hunk 1 above)
Comment (stifoon) [inline documentation/slate/source/includes/apis.md:1050]: Agree with TeXitoi for the "see description in journeys".
And that way, you won't have to correct the traveler_type bug in documentation ;)

File: documentation/slate/source/includes/apis.md (hunk 2 above)
Comment (stifoon) [inline documentation/slate/source/includes/apis.md:1060]: You should also leave a  "see description in journeys" there.

File: documentation/slate/source/includes/apis.md (hunk 3 above)
Comment (stifoon) [inline documentation/slate/source/includes/apis.md:1066]: Again, a "see description in journeys" there.

**Pull request closed without merging**